src/CompKodiProps.cpp

```diff
@@ -15,6 +15,7 @@
 #include <rapidjson/document.h>
 
 #include <string_view>
+#include <decrypters/Helpers.h>
```

Comment:
and moved above by sorting it
```suggestion
#include "decrypters/Helpers.h"
```